be/src/connector_sink/hive_chunk_sink.cpp

```diff
@@ -41,6 +42,7 @@ HiveChunkSink::HiveChunkSink(std::vector<std::string> partition_columns,
 
 Status HiveChunkSink::init() {
     RETURN_IF_ERROR(ColumnEvaluator::init(_partition_column_evaluators));
+    RETURN_IF_ERROR(_file_writer_factory->init());
```

Comment:
FileWriter::commit and write seems blocking operations, it must be non-blocking.

Reply:
non-blocking